package.json

```diff
@@ -142,7 +142,7 @@
     "test-client:watch": "nodemon -e js,jsx --exec npm run test-client",
     "test-server:watch": "nodemon -e js,jsx --exec npm run test-server",
     "test-test:watch": "nodemon -e js,jsx --exec npm run test-test",
-    "lint": "bin/run-lint",
+    "lint": "bin/run-lint .",
```

Comment:
To make `npm run lint` run as before despite the changes I have made to `run-lint` script.  See ![image of changes to run-lint script](https://cloud.githubusercontent.com/assets/19974613/16744098/c0b1d4fc-47af-11e6-9c9d-935d132b390a.png).

As the purpose of this PR is to run `eslint` only for files modified within the branch (see rationale in the comment above), what happens when the branch does not modify any JS/JSX file and the result of the git expression is void? Current `run-lint` take that as an indication that it should run for the whole codebase. But, what we actually want in this specific case (at least in CircleCI) is to prevent eslint from running.